.gitignore

```diff
@@ -29,6 +31,9 @@ server/dist/
 ssr/node_modules/
 ssr/dist/
 watcher/node_modules/
+archivecontent/
 content/node_modules/
+content/coverage/
+content/files/
```

Comment:
I was hoping, that as soon as we get a somewhat decent ks renderer, that we can actually check in all the `content/files/**/index.html` and `content/files/**/index.yaml` files for good. 
But perhaps we can do that as a final hurray the day this PR merges into master.

Reply:
Sounds good!